app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1> Welcome to your to-do list! Your tasks are: </h1>
+
+<div class = "tasklist">
+  <% @task_list.each do |task| %>
+  <% if @task_list == nil %>
```

Comment:
Ugh, I can't figure out how to reply to your very last overall comment, but I wanted to sincerely thank you for taking the time to review my code. Your comments will really help me learn and grow, and I really, truly appreciate it.